src/ai_marketplace_monitor/telegram.py

```
@@ -0,0 +1,186 @@
+            elif self.message_format == "markdown":
+                return f"*{md_escape(text)}*"
+            else:
+                return text
+
+        def format_italic_link(text: str, url: str) -> str:
+            if self.message_format == "html":
+                return f'<i><a href="{html_escape(url)}">{html_escape(text)}</a></i>'
+            elif self.message_format == "markdown":
+                return f"_[{md_escape(text)}]({url})_"
+            else:
+                return f"{text}: {url}"
```

Pylint (3.3.7)

[refactor] Unnecessary "elif" after "return", remove the leading "el" from "elif"

(R1705)